src/test/resources/db/migration/V1__init_db.sql

```diff
@@ -58,64 +60,73 @@ create unique index new_digital_media_object_id_version_url
+    last_checked                 timestamptz NOT NULL,
+    deleted                      timestamptz NULL,
+    "data"                       jsonb NULL,
+    original_data                jsonb NULL,
+    dwca_id                      text NULL,
+    CONSTRAINT new_digital_specimen_pk PRIMARY KEY (id)
 );
 CREATE INDEX new_digital_specimen_created_idx ON public.new_digital_specimen USING btree (created);
 CREATE INDEX new_digital_specimen_id_idx ON public.new_digital_specimen USING btree (id, created);
 CREATE INDEX new_digital_specimen_physical_specimen_id_idx ON public.new_digital_specimen USING btree (physical_specimen_id);
 
-
-CREATE TABLE public.organisation_do (
-	id text NOT NULL,
-	organisation_name text NULL,
-	organisation_code text NULL,
-	city text NULL,
-	country text NULL,
-	country_code text NULL,
-	"data" jsonb NULL,
-	CONSTRAINT organisation_do_pkey PRIMARY KEY (id)
-);
-
-CREATE TABLE machine_annotation_services
+CREATE TABLE public.machine_annotation_services
 (
-    id text not null
+    id                            text                     not null
         primary key,
-    version integer not null,
-    name varchar not null,
-    created timestamp with time zone not null,
-    administrator text not null,
-    container_image text not null,
-    container_image_tag text not null,
+    version                       integer                  not null,
+    name                          varchar                  not null,
+    created                       timestamp with time zone not null,
+    administrator                 text                     not null,
+    container_image               text                     not null,
+    container_image_tag           text                     not null,
     target_digital_object_filters jsonb,
-    service_description text,
-    service_state text,
-    source_code_repository text,
-    service_availability text,
-    code_maintainer text,
-    code_license text,
-    dependencies text[],
-    support_contact text,
-    sla_documentation text,
-    topicname text,
-    maxreplicas integer,
-    deleted_on timestamp with time zone
-);
\ No newline at end of file
+    service_description           text,
+    service_state                 text,
+    source_code_repository        text,
+    service_availability          text,
+    code_maintainer               text,
+    code_license                  text,
+    dependencies                  text[],
+    support_contact               text,
+    sla_documentation             text,
+    topicname                     text,
+    maxreplicas                   integer,
+    deleted_on                    timestamp with time zone
+);
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+create table mas_job_record
+(
+    job_id         uuid default uuid_generate_v4() not null
+        constraint mas_job_record_pk
+            primary key,
+    state          text                            not null,
```

Comment:
Could use postgres enum for this. https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
so for some reason this needs to stay "text" type but the db has this enum, and also it's reflected in the jooq codegen